pulser/parametrized/paramobj.py

```diff
@@ -18,10 +18,11 @@
 import inspect
 import operator
 import warnings
-from typing import Callable, Dict, Any, Union
+from typing import Callable, Dict, Any, Union, TYPE_CHECKING
 
-from pulser.parametrized import Parametrized, Variable
 from pulser.json.utils import obj_to_dict
+if TYPE_CHECKING:
+    from pulser.parametrized import Parametrized, Variable
```

Comment:
You can't really do this because `Parametrized` is needed even when you're not type checking. You have to split the import into two lines

Reply:
Updated